Pipfile

```diff
@@ -38,6 +38,7 @@ environs = {extras = ["django"], version = "*"}
 hfst-optimized-lookup = "*"
 tqdm = "~=4.40"
 whitenoise = "*"
+foma = {subdirectory = "foma/python", git = "https://github.com/andrewdotn/foma"}
```

Comment:
Is this git repo url what we want here? Does it matter that it's pointing to one of your repos?

Reply:
The Foma Python binding is not available on PyPI so Andrew made a fork to make it work.